packages/CalendarCarousel/index.tsx

```diff
@@ -57,48 +60,65 @@ const styles = StyleSheet.create<Style>({
     textAlign: 'center',
     justifyContent: 'center',
   },
-  nextArrowText: {
-    color: 'royalblue',
-    fontSize: 30,
-  },
-  weekdayFlex: {
+  weekdayStyle: {
     flexDirection: 'row',
   },
   weekdayText: {
     color: '#4F4F4F',
     fontSize: 20,
     textAlign: 'center',
   },
-  calendarContainerSize: {
+  dayContainer: {
     width: 330,
     height: 388,
   },
-  noPressView: {
+  otherDaysView: {
     width: 47,
     height: 47,
     paddingTop: 13.5,
+  },
+  otherDaysText: {
+    color: 'lightgray',
+    textAlign: 'center',
+  },
+  currentDayView: {
+    borderRadius: 50,
+    backgroundColor: '#109CF1',
+    width: 47,
+    height: 47,
     alignItems: 'center',
+    paddingTop: 13.5,
   },
-  noPressText: {
+  currentDayText: {
+    color: 'white',
     textAlign: 'center',
   },
-  onPressView: {
+  notActiveView: {
+    width: 47,
+    height: 47,
+    paddingTop: 13.5,
+    alignItems: 'center',
+  },
+  notActiveText: {
+    textAlign: 'center',
+  },
+  activeView: {
     width: 47,
     height: 47,
     paddingTop: 13.5,
     alignItems: 'center',
     borderRadius: 50,
     backgroundColor: '#F0F8FD',
   },
-  onPressText: {
+  activeText: {
     textAlign: 'center',
     color: '#109CF1',
   },
 });
 
 interface Props<T> {
   date?: Date;
-  onDateChanged?: (date: Date) => void;
+  onMonthChanged?: (date: Date) => void;
```

Comment:
Why changed to `onMonthChanged` when passing the date?

Reply:
I thought it would be a little bit more specific to say Month Change because the function is changing the month. So I was experimenting with Daniel. I forgot to change it back

Reply:
Looks like you've not changed this back yet!